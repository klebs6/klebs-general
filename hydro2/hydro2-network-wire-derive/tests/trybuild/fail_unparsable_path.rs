// ---------------- [ File: hydro2-network-wire-derive/tests/trybuild/fail_unparsable_path.rs ]
// tests/trybuild/fail_unparsable_path.rs

use hydro2_network_wire_derive::NetworkWire;

#[derive(NetworkWire)]
#[available_operators(
   op="SomeOp("  // This is invalid path syntax
)]
pub struct MyWire;

fn main() {}
