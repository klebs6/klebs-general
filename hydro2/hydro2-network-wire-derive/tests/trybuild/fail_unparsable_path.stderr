error: Could not parse `SomeOp(` as a Path.
 --> tests/trybuild/fail_unparsable_path.rs:8:7
  |
8 |    op="SomeOp("  // This is invalid path syntax
  |       ^^^^^^^^^
